test/unit/utils/test_common.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

Comment:
```suggestion
# Copyright (c) Fairlearn contributors.
```
This will work with our updated flake8 setup.

Reply:
but that PR is not merged.

Reply:
It is merged already (I bundled it with the deprecation PR).

Reply:
https://github.com/fairlearn/fairlearn/blob/1247b081b5cbdf9373595b7e4627c142ae30ae44/setup.cfg#L6

Reply:
> It is merged already (I bundled it with the deprecation PR).

That's generally a bad idea though. It's much nicer when things are in their own PR, for traceability and future reference.

Reply:
i know... i was too lazy... but i'll do it properly next time!